common/display/virtualdisplay.cpp

```diff
@@ -420,6 +420,10 @@ void VirtualDisplay::SetOutputBuffer(HWCNativeHandle buffer,
   }
 #endif
 
+  if (output_handle_ && output_handle_ == buffer) {
```

Comment:
This situation is same with the negation of next if statement. It is better adding a else statement instead of add a new if segment.